src/gc/env/gcenv.object.h

```diff
@@ -64,19 +66,31 @@ class MethodTable
         return m_componentSize;
     }
 
+    bool Collectible()
```

Comment:
A nit - it seems that it would be better to name the function `IsCollectible` (and modify the define accordingly) to match the other flags that are in a form of Verb + Noun.